pkg/kubelet/container_bridge.go

```diff
@@ -122,24 +140,26 @@ func cbr0Exists() (bool, error) {
 	return true, nil
 }
 
-func cbr0CidrCorrect(wantCIDR *net.IPNet) bool {
-	output, err := exec.Command("ip", "addr", "show", "cbr0").Output()
-	if err != nil {
+func cbr0CidrCorrect(br *netlink.Bridge, wantCIDR *net.IPNet) bool {
+
+	addrs, err := netlink.AddrList(br, syscall.AF_INET)
+	if err != nil && err != syscall.ENOENT {
+		glog.Errorf("could not get list of IP addresses from bridge %q: %v", br.Name, err)
 		return false
 	}
-	match := cidrRegexp.FindSubmatch(output)
-	if len(match) < 2 {
+
+	// TODO: Document why two addresses are required, and why the second one is chosen
```

Comment:
This is odd; I don't think the code should rely on the number or ordering of addresses.  What is the other address?